apps/tome-api/src/features/card/card.fns.ts

```diff
@@ -750,6 +775,173 @@ export const deck: DbCard[] = [
 			},
 		},
 	},
+	{
+		id: '55',
+		name: 'Burning flame',
+		type: 'spell',
+		colors: ['red'],
+		attack: 18,
+		description:
+			'Every time this spell deals damage, discard the bottom card from its stack. If there’s no more cards to discard, discard Burning flame.',
+		effects: {
+			onDealDamage: async function* ({ actions, game, ownerSide, thisCard }) {
+				const stack = STACKS.find(stack => game.board.players[ownerSide].stacks[stack].includes(thisCard));
+				if (!stack) {
+					yield* actions.discard({ card: thisCard, from: game.board.field, side: ownerSide });
+					return;
+				}
+				const cardToDiscard = game.board.players[ownerSide].stacks[stack].pop();
```

Comment:
`pop` is a destructive method that'll get the top one, so this will actually represent the card itself (Burning flame) and will already remove it, so it'll be quite quirky.

look at `topOf` and engineer a `bottomOf` instead! so we can call `bottomOf(stack)`

Reply:
Of course! I for some reason convinced myself that the stacks were in reverse order. I think I was just being silly.